docs/software/ssh.md

```diff
@@ -2,12 +2,71 @@
 
 ## Overview
 
-- [Dropbear - Lightweight SSH Server](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p62)  
-- [OpenSSH - Feature rich SSH server with SFTP/SCP support.](https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=10#p63)  
+- [**Dropbear - Lightweight SSH Server**](#dropbear-lightweight-ssh-server)  
+- [**OpenSSH - Feature rich SSH server with SFTP/SCP support**](#openssh-feature-rich-ssh-server-with-sftpscp-support)  
+
+??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
+    To install any of the **DietPi Optimised Software** listed below run from the command line:
+    ```
+    dietpi-launcher
+    ```
+    and selected _DietPi-Software_. or launch directly the tool:
+    ```
+    dietpi-software
+    ```
+
+    Choose **Software optimised** and select one or more items. Finally click on `Install`. DietPi will do all the necessary steps to install and start these software items.
+
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
 
-!!! warning "Documentation in progress..." 
+## Dropbear - Lightweight SSH Server
+
+Dropbear is a lightweight SSH server, installed by default on DietPi systems.  
+Remark: You can swap or change your SSH server at any time using *DietPi-Software*.
+
+![DietPi ssh server software Dropbear](../assets/images/dietpi-software-sshserver-dropbear.jpg){: style="width:150px"}
+
+### Access to the SSH server
+
+The SSH server can be connected directly from Unix with the `ssh` command line program or other SSH client programs:
+
+- address = Your IP address (e.g.: *192.168.0.100*)
+- Port = *22*
+- Username = *root*
+- Password = *dietpi*
+
+### Windows SSH client
+
+Windows users can download the ready to run DietPi PuTTY/KiTTY SSH Client.
```

Comment:
Lets make this links, same below for OpenSSH:
```
On Windows, [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) and [KiTTY](http://www.9bis.net/kitty/#!pages/download.md) are recommended SSH clients.
```
On Windows, [PuTTY](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html) and [KiTTY](http://www.9bis.net/kitty/#!pages/download.md) are recommended SSH clients.

Regarding
> ready to run DietPi PuTTY/KiTTY SSH Client

I remember there was a KiTTY download on our server, of course badly outdated, with `DietPi` profile and `192.168.0.100` IP pre-configured. However, der IP needs to be adjusted anyway and I'd personally never download 3rd party Windows software from a non-original source, as general good practice, hence I replaced that link with the official KiTTY download and removed that one from our server. However, while KiTTY has some nice feature, I personally stick with the only true original: PuTTY, hence at least link both 😄.

Reply:
Changed.  
I also only use PuTTY (or ssh user@host under Linux, of course).